README.md

```diff
@@ -1,36 +1,83 @@
 # feedforward
 
-This library allows you to apply steps towards a goal without specifying an
-explicit DAG and still getting what I'll call "optimistic parallelism."  If
-you're familiar with branch prediction, this is the same basic idea applied to
-arbitrary key-value environments.
+This is a dataflow-esque library that allows you to do transforms on items in
+one direction towards a goal.  Where it differs from other dataflow models is
+that there is only `map` and the items can never never change type (for a given
```

Comment:
Similar to a comment I make below: I'm not sure if the reader is well-versed in existing approaches to the problem, and so might not be able to appreciate comparisons like this.

Reply:
I was trying to get rid of the "without" in the previous version... what about saying it's "an efficient way to run a linear DAG"